### ISSUE TITLE
[BUG] Content of notes created by case templates is not visible - template create of notes not complete

### ISSUE THREAD

**Issue by StaSys846** [dfir-iris/iris-web#380]
**Describe the bug**
Notes content is not visible via WebUI when they are created by case templates. 
Note groups and notes are created correctly and are visible under 'Notes' but click on single notes to get the content leads to HTTP/1.1 400 Bad request.

Reason: Fields in notes db schema are not filled appropriately and important fields are missing for a note in the database (e.g. case_id). Adding proposed lines of code (see additional context) fixes the issue.
iris v2.4.1

**To Reproduce**
1. Go to 'Create Case' and choose a case template with notes
2. Click on 'Create'
3. Switch to newly created case and go to 'notes'
4. Select single note to view content -> No error in WebGUI but 400 Bad request in the background

**Expected behavior**
Notes are created consistently during case creation with case templates and notes content is visible by single click on notes.

**Screenshots**
<img width="324" alt="image" src="https://github.com/dfir-iris/iris-web/assets/140630979/01ee940c-f001-4a2c-9577-1e7896095f10">

**Desktop (please complete the following information):**
 - OS: independent
 - Browser independent
 - Version independent

**Smartphone (please complete the following information):**
 - independent

**Additional context**
Following note fields are not filled during template case create in source/app/datamgmt/manage/manage_case_templates_db.py:
- note_creationdate
- note_lastupdate
- note_user
- note_case_id

Especially missing note_case_id in the database leads to the error during notes content loading via the web UI

Proposal:
Adding the following lines of code in source/app/datamgmt/manage/manage_case_templates_db.py in function 210:case_template_populate_notes() starting at line 229:
```
note.note_creationdate = datetime.utcnow()
note.note_lastupdate = datetime.utcnow()
note.note_user = case.user_id
note.note_case_id = case.case_id
```
<img width="840" alt="image" src="https://github.com/dfir-iris/iris-web/assets/140630979/974d4969-7084-4178-8498-07a3084e063a">

**Comment by whikernel** [comment]
Fixed in v2.4.2. Thanks a again... definitely some big oversights here

**Comment by StaSys846** [comment]
You're welcome and many thanks for the ultra fast fixes/code updates
